CHANGELOG.md

```diff
@@ -24,6 +24,8 @@
     - Added `get_field` to query fields of structured dtypes
     - Additional helper methods: `has_subarray`, `has_fields`, `is_native_byteorder`
     - Renamed `get_type` to `typeobj`
+    - `new` method, equivalent to invoking the constructor
```

Comment:
If this gets the honours of a link to the PR, then line 21 does seem to deserve it as well?

Reply:
I am sorry for the snark. It does not belong in a code review. Not the best of Monday mornings today.

Reply:
No worries, that's what Monday mornings are there for 😄 I agree this looks a bit inconsistent in the level of detail and references to PRs, will take a look at cleaning it up

Reply:
Ok, I've tried to shorten the changelog entries for all recent dtype-related changes as much as possible, but I still think we should note it if we change or remove something public (e.g. public type removed).